pubsub/tests/unit/pubsub_v1/subscriber/test_subscriber_client.py

```diff
@@ -54,3 +54,13 @@ def test_subscribe_with_callback():
         subscription = client.subscribe('sub_name_b', callback)
         open_.assert_called_once_with(callback)
     assert isinstance(subscription, thread.Policy)
+
+
+def test_subscribe_with_failed_callback():
+    import pytest
+    creds = mock.Mock(spec=credentials.Credentials)
+    client = subscriber.Client(credentials=creds)
+    callback = 'abcdefg'
+    with pytest.raises(TypeError) as exception:
+        subscription = client.subscribe('sub_name_b', callback)
+    assert callback in str(exception.value)
```

Comment:
:+1: For this check. I'd suggest renaming `exception` to `exc_info` (it's not actually an exception, it's some context manager provided by `pytest`)